.github/workflows/fc-asciidoctor-publish.yaml

```diff
@@ -0,0 +1,49 @@
+name: fc-asciidoctor-publish
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+    paths:
+      - utilities/fc-asciidoctor/version.json
+      - .github/workflows/fc-asciidoctor-publish.yaml
+jobs:
+  build:
+    env:
+      context: utilities/fc-asciidoctor
+      image_name: fc-asciidoctor
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@master
+      - name: Get image tags
+        id: image_tags
+        run: |
+          echo -n ::set-output name=IMAGE_TAGS::
+          VERSION=$(jq -r '.version' ${context}/version.json)
+          TAGS=('latest')
+          if [ "${VERSION}" ] && [ "${VERSION}" != "latest" ]; then
+              TAGS+=("${VERSION}")
+          fi
+          if [[ "${GITHUB_REF}" =~ refs/tags/(.*) ]]; then
+              TAGS+=("git-${BASH_REMATCH[1]}")
+          fi
+          echo "${TAGS[*]}"
+      - name: Build image
+        id: build_image
+        uses: redhat-actions/buildah-build@v2
+        with:
+          context: ${{ env.context }}
+          dockerfiles: |
+            ./${{ env.context }}/Dockerfile
+          image: ${{ env.image_name }}
+          tags: "${{ steps.image_tags.outputs.IMAGE_TAGS }}"
+      - name: Push to Quay
```

Comment:
i'd add in the ghcr push as well, like:
- https://github.com/redhat-cop/containers-quickstarts/blob/master/.github/workflows/jenkins-agent-python-publish.yaml#L33-L51

allows forks to then test a built image with them needing access or setup of quay creds.